internal/slicer/slicer.go

```diff
@@ -291,16 +314,30 @@ func Run(options *RunOptions) (*Report, error) {
 		}
 		err := scripts.Run(&opts)
 		if err != nil {
-			return nil, fmt.Errorf("slice %s: %w", slice, err)
+			return fmt.Errorf("slice %s: %w", slice, err)
 		}
 	}
 
 	err = removeAfterMutate(targetDir, knownPaths)
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	return report, nil
+	pkgInfos := []*archive.PackageInfo{}
+	for pkg, _ := range packages {
+		pkgInfo, err := archives[pkg].Info(pkg)
```

Comment:
See earlier note about re-selecting all packages again here.

Reply:
Addressed in the thread https://github.com/canonical/chisel/pull/142#discussion_r1771654141.